druid/src/widget/list.rs

```diff
@@ -117,6 +117,69 @@ impl<T: Data> ListIter<T> for Vector<T> {
     }
 }
 
+#[cfg(feature = "im")]
+impl<K, V> ListIter<V> for OrdMap<K, V>
+where
+    K: Data + Ord,
+    V: Data,
+{
+    fn for_each(&self, mut cb: impl FnMut(&V, usize)) {
+        for (i, item) in self.iter().enumerate() {
+            let ret = (item.0.to_owned(), item.1.to_owned());
+            cb(&ret.1, i);
+        }
+    }
+
+    fn for_each_mut(&mut self, mut cb: impl FnMut(&mut V, usize)) {
+        for (i, item) in self.clone().iter().enumerate() {
+            let mut ret = (item.0.clone(), item.1.clone());
```

Comment:
I don't think we need to clone the key here? I think we can just use the key from `item`.

Reply:
Yes, the clone looks unnecessary. I'm guessing it snuck in because I wrote the key and value implementation and then copy-pasted.